src/main/java/com/instana/operator/AgentDeployer.java

```diff
@@ -356,6 +347,23 @@ DaemonSet newDaemonSet(InstanaAgent owner,
     return daemonSet;
   }
 
+  private void configureOpentelemetry(Container container, InstanaAgentSpec config) {
+    // Get ImagePullPolicy value
+    String otelActiveFromEnvVar = environment.get(Environment.RELATED_INSTANA_OTEL_ACTIVE);
+    String otelPortFromEnvVar = environment.get(Environment.RELATED_INSTANA_OTEL_PORT);
+
+    Boolean otelActiveFromCustomResource = config.getAgentOtelActive();
+    Integer otelPortFromCustomResource = config.getAgentOtelPort();
+
+    if(otelActiveFromCustomResource || getBoolean(otelActiveFromEnvVar)){
+      Integer otelPort = otelPortFromCustomResource;
+      if(isInteger(otelPortFromEnvVar)){
+        otelPort = getInteger(otelPortFromEnvVar);
+      }
+      container.getPorts().add(new ContainerPort(otelPort,null, null,null,null));
```

Comment:
On the Agent-side, the OTel port is fixed. So while we could expose this under a different port-number, the `containerPort` should always be the same.
I'll see if we can successfully change the `hostPort` though, so someone could expose it differently.

Reply:
should i remove this feature for the meanwhile ?
I don't think that we really need it.

Reply:
Yes, I believe that would be easier for now if you don't need it.

Reply:
Okay, i am going to remove it..
Probably I will need to create a new PR, since i need to get rid of the old commits.

Reply:
Ok, its done. I reduced it to 2 commits. One Commit per support Ticket.